csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -94,45 +94,53 @@ namespace vllm {
 template <typename scalar_t, typename scale_type>
 __global__ void static_scaled_int8_quant_kernel(
     scalar_t const* __restrict__ input, int8_t* __restrict__ out,
-    scale_type const* scale_ptr, const int hidden_size) {
+    scale_type const* scale_ptr, const int64_t hidden_size) {
```

Comment:
I think `hidden_size` should still be an `int`, otherwise e.g. `i` may be up-converted for comparison